loader/src/ui/nodes/TextAreaV2.cpp

```diff
@@ -0,0 +1,377 @@
+#include <Geode/ui/TextAreaV2.hpp>
+#include <Geode/DefaultInclude.hpp>
+
+using namespace geode::prelude;
+
+#define IMPL_GETTER(type, name, methodName) \
+    type geode::TextAreaV2::get##methodName() const { return m_##name; }
+#define IMPL_GETTER_SETTER(type, paramType, name, methodName) \
+    void geode::TextAreaV2::set##methodName(const paramType name) { m_##name = name; this->update(); } \
+    IMPL_GETTER(type, name, methodName)
+
+geode::TextAreaV2* geode::TextAreaV2::create(const std::string& text, const std::string& font, const float scale, const float width, const float height, const bool deferUpdates) {
+    TextAreaV2* area = new TextAreaV2(text, font, scale, width, height, deferUpdates);
+
+    if (area && area->init()) {
```

Comment:
if (area) is pointless, area will never be nullptr
either use std::nothrow or dont bother to check it (geode does the latter)